raft/src/main/java/org/apache/kafka/raft/metadata/MetaLogRaftShim.java

```diff
@@ -124,6 +126,15 @@ public void handleCommit(BatchReader<ApiMessageAndVersion> reader) {
             }
         }
 
+        @Override
+        public void handleSnapshot(SnapshotReader<ApiMessageAndVersion> reader) {
+            // TODO: Create Jira: Handle loading commit in ListenerShim
```

Comment:
We can probably take care of this when we merge the two interfaces. I'm hoping to get to that shortly.